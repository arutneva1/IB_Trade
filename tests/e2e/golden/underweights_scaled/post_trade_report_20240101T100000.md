| symbol | side | filled_shares | avg_price | notional | avg_slippage | residual_drift_bps |
| --- | --- | --- | --- | --- | --- | --- |
| BBB | BUY | 1.0000 | 51.00 | 51.00 | 0.00 | -10.06 |
| AAA | BUY | 1.0000 | 101.00 | 101.00 | 0.00 | -60.36 |
