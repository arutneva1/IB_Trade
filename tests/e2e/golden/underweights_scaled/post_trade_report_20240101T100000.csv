symbol,side,filled_shares,avg_price,notional,residual_drift_bps
BBB,BUY,1.0,51.0,51.0,-10.06
AAA,BUY,1.0,101.0,101.0,-60.36
